arduino/walt/walt.ino

```diff
@@ -39,6 +39,7 @@
 
 #define CMD_AUDIO               'A'
 #define CMD_BEEP                'B'
+#define CMD_BEEP_STOP           'S'
```

Comment:
It would be good if there was a single common source for the .java and .ino (.c) definitions